noiseworks/settings.py

```diff
@@ -273,3 +273,31 @@
 file_storage_relative_path = env.str("FILE_STORAGE_RELATIVE_PATH", None)
 if file_storage_relative_path:
     MEDIA_ROOT = BASE_DIR / file_storage_relative_path
+
+# Logging
+
+LOGGING = {
+    "version": 1,
+    "disable_existing_loggers": False,
+    "formatters": {
+        "timestamp": {
+            "format": "{asctime} {levelname} {message}",
+            "style": "{",
+        },
+    },
+    "handlers": {
+        "file": {
+            "level": env("LOG_LEVEL"),
+            "class": "logging.FileHandler",
+            "filename": env("LOG_FILE"),
```

Comment:
This one is a bit trickier - is there some default path that we can derive from `BASE_DIR` maybe?